background-remover/background-remover.py

```diff
@@ -0,0 +1,55 @@
+import os, glob
+import rembg
+
+def process (input_path: str, output_path: str, session: rembg.bg.BaseSession) -> bool:
+    print(f"making file at {input_path} to {output_path}")
+    if not os.path.exists(input_path):
+        print(f"ERROR: Process failed due to input file does not exist [{input_path}]")
+        return False
+        
+    output_dir_path = os.path.dirname(output_path)
+    if not os.path.isdir(output_dir_path):
+        print(f"LOG: Process creates output directory [{output_dir_path}].")
+        os.makedirs(output_dir_path)
+
+    try:
+        with open(input_path, 'rb') as i:
+            input = i.read()
+
+        output = rembg.remove(input, session=session)
+
+        with open(output_path, 'wb') as o:
+            o.write(output)
+
+    except Exception as error:
+        print(f"ERROR: Process failed due to exception [{error}]")
+        return False
+
+    return True
```

Comment:
I believe it would be better for the exception to leave this function (and be annotated with extra info if possible). Boolean alone doesn't convey enough information on it's own

Reply:
i am not a fan of throwing exceptions, should we make it an enum?

Reply:
it becomes hard to know which method should handle what exception, and you can end up with chains of exceptionhandlings. Which can become really convoluted to understand

Reply:
it's okay to convert the exception to error values, though you will need to handle it correctly yourself